docs/articles/documentation/examples/README.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: docs
+title: Examples
+permalink: /documentation/examples/
+---
+# Examples
+
+This section lists examples from the [testcafe-examples](https://github.com/DevExpress/testcafe-examples) repository. All examples are ready to run.
+
+## Locating Elements
+
+Example | Description
+------- | ------------
+[Find Element By Trimmed Text](https://github.com/DevExpress/testcafe-examples/tree/master/examples/find-element-by-trimmed-text) | Find a header element with extra space characters by trimmed text content.
```

Comment:
Can you please clarify the functionality of this feature - for my understanding?
Does this feature search for elements that have additional space characters within the website/web page that is being tested?

Reply:
No, in this example, the page header consists of a meaningful part and some random number of space characters around it. We know the meaningful part (i.e. the "trimmed text"), so this example shows how we can find the header element using this part only, without knowing anything about these random space characters.

So, we can say something like:

*Shows how to ignore white-space characters when searching for an element by text.*
*Finds an element by text without regard to white-space characters at its ends.*

Reply:
```suggestion
[Find Element By Trimmed Text](https://github.com/DevExpress/testcafe-examples/tree/master/examples/find-element-by-trimmed-text) | Ignores white-space characters when searching for an element by text.
```